README.md

```diff
@@ -209,8 +209,12 @@ Override the following settings:
 * The extraction of `shorteners.txt` and performance improvements to the
   shortener list were contributed by [Avner
   Cohen](https://github.com/AvnerCohen);
-* The default list of shorteners comes from [LongURL.org's curated
-  list](http://longurl.org/services).
+* The default list of shorteners includes:
+    * [LongURL.org's](http://longurl.org/services) curated list
```

Comment:
I thought it was only 2 spaces, but my preview wasn't showing the sublist when I first tried it...
Anyway, it's updated (and it's working with 2 spaces too, so it was my preview).